packages/gradbench/src/App.tsx

```diff
@@ -1,10 +1,151 @@
+import "./App.css";
+
 const App = () => {
   return (
     <>
-      <h1>GradBench</h1>
+      <div className="header-cont">
+        <h1 className="header">
+          GradBench
+          <span className="subtitle">
+            An Automatic Differentiation Benchmarking Suite
```

Comment:
Can we replace this with something a bit more descriptive/catchy? Maybe use the tagline in the "about" section of the GitHub repo:

```suggestion
            a benchmark for differentiable programming across languages and domains
```

And I'd probably make it lowercase so it feels more like a description and less like a subtitle.